Gopkg.toml

```diff
@@ -31,10 +31,18 @@ required = [
   "github.com/docker/distribution/reference",
   "k8s.io/api/core/v1",
   "k8s.io/apimachinery/pkg/apis/meta/v1",
-  "k8s.io/apiserver/pkg/endpoints/request"
+  "k8s.io/apiserver/pkg/endpoints/request",
+  "github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
```

Comment:
`apiextensions` isn't needed. The required block is only needed when a package isn't used in our code.